tests/integration/test_client.py

```diff
@@ -217,7 +220,7 @@ async def test_empty_string(client: Client, table: TableName):
 
 
 async def test_empty_item(client: Client, table: TableName):
-    with pytest.raises(EmptyItem):
+    with pytest.raises(ValidationException):
```

Comment:
Does it know that `h` and `r` are special? 🤔

Reply:
thee client doesn't. the server does. `ValidationException` comes from the server.